artillery/docker-compose.yml

```diff
@@ -86,5 +88,12 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
       - API_BASE_URL_OAUTH=http://hmpps-auth:8080/auth
 
+  prisoner-search:
+    image: quay.io/hmpps/ppud-fake-prisoner-offender-search-api:latest
```

Comment:
Not sure I understand the benfit of using their stubs when we could just use Wiremock ourselves?